client/src/services/auth.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios"
+
+const baseUrl = "/api/auth"
+
+export const createNewUser = async (newUser) => {
+  try {
+    const response = await axios.post(`${baseUrl}/email/register`, { user_info: newUser })
+    console.log(response.data)
+  } catch (err) {
+    console.log(err.message)
+  }
+}
+
+export const loginUser = async (credentials) => {
+  try {
+    const response = await axios.post(`${baseUrl}/login`, credentials)
```

Comment:
If this is email login, it should be  await axios.post(`${baseUrl}/email`, credentials)

Reply:
Though I could change it to api/auth/email/login, if that would be more straightforward